src/components/AlertCard/index.js

```diff
@@ -10,90 +10,86 @@ import classNames from "classnames"
 import PropTypes from "prop-types"
 import ReactMarkdown from "react-markdown"
 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faAngleDown, faAngleRight } from "@fortawesome/free-solid-svg-icons"
+
 import styles from "./styles.module.css"
 
 const AlertCard = props => {
   const severityMap = {
-    0: "Fyi Severity",
-    1: "Low Severity",
-    2: "Moderate Severity",
-    3: "High Severity",
+    0: "Severity: Fyi",
+    1: "Severity: Low",
+    2: "Severity: Moderate",
+    3: "Severity: High",
   }
-  const sevsme = props.severity
-  const sevClassName = severityMap[props.severity].split(" ")[0]
-  // const alerts = props.alerts.filter(item => item)
-  // const alertLength = alerts.length
-  // const highSeverity = props.predictions.alerts.severity === 3
-  // const midSeverity = props.predictions.alerts.severity === 2
-  // const lowSeverity = props.predictions.alerts.severity === 1
-  // const fyi = props.predictions.alerts.severity === 0
 
-  // console.log("props", props.content)
+  const [toggle, setToggle] = useState(false)
+  const toggleCaret = () => setToggle(!toggle)
+  const sevClassName = severityMap[props.severity].split(": ")[1]
```

Comment:
If we use [toLowerCase](ctm1989@gmail.com) this should solve the class naming issue.

Reply:
not understanding what this means

Reply:
This was referencing the fact that in the styling for the alert card, there are classes named `.Low` and `.Med`, which should be lower case, not title case.